pkg/iauthnzimpl/impl_acl.go

```diff
@@ -59,6 +59,15 @@ var defaultACL = ACL{
 		},
 		policy: ACPolicy_Allow,
 	},
+	{
```

Comment:
better approach:
```golang
{
	desc: "allowed to sys.Guest login, i.e. without principal token at all",
	pattern: PatternType{
		principalsPattern: [][]iauthnz.Principal{{{Kind: iauthnz.PrincipalKind_User, WSID: istructs.GuestWSID}}},
		qNamesPattern: []appdef.QName{
			qNameCmdProvideCertificatePart, qNameCmdProvideCertificate, qNameQryGetCustomerStatus,
			qNameCmdFiscalizeDocument, qNameQryFiscalizationResultStatus, qNameCmdCreateExport, qNameQryExportStatus},
	},
        policy: ACPolicy_Allow,
},
```
note: a valid token is provided -> sys.Guest login is not added among principals -> this rule will not work. Other words, this rule is only for the case when there is no principal token at all.